### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by abramsymons**
- Enable nodes not having a context to process onchain sponsor requests for apps of that context
- Add a verification for each app
- Remove outdated ethName from contexts

### CODE REVIEW

File: web_services/foxx/brightid/initdb.js [hunk 1]
```diff
@@ -206,7 +206,20 @@ function v5_5() {
...
 
-const upgrades = ['v5', 'v5_3', 'v5_5'];
+function v5_6() {
+  console.log("removing 'verification' attribute form contexts collection");
+  const contextsColl = arango._collection('contexts');
+  query`
+    FOR doc IN ${contextsColl}
+      REPLACE doc WITH UNSET(doc, 'verification') IN ${contextsColl}`;
+
+  console.log("removing 'ethName' attribute form context collection");
```
Comment (abramsymons) [inline web_services/foxx/brightid/initdb.js:216]: `const ETHNAME_NOT_SET = 8;` should also be removed from `index.js` file

File: updater/sponsorships.py [hunk 2]
```diff
@@ -54,44 +58,49 @@ def update():
...
-            collection = context['collection']
-            c = db[collection].find(
-                {'contextId': context_id})
-            if c.empty():
-                print("the context id doesn't link to any user under this context")
-                continue
-            user = c.next()['user']
 
             # remove testblocks if exists
             testblocks.delete_match({
```
Comment (abramsymons) [inline updater/sponsorships.py:65]: It seems that we can remove this section as the node will apply that when `Sponsor` operation gets received.

File: updater/sponsorships.py [hunk 3]
```diff
@@ -54,44 +58,49 @@ def update():
...
-              'contextId': context_id,
-              'action': 'sponsorship',
-              'app': app['_key']
+                'contextId': context_id,
+                'action': 'sponsorship',
+                'app': app['_key']
             })
 
-            c = sponsorships.find(
```
Comment (abramsymons) [inline updater/sponsorships.py:76]: This makes sense we keep this check and not waste gas to send `Sponsor` operation for already sponsored user.

File: web_services/foxx/brightid/initdb.js (hunk 1 above)
Comment (siftal) [reply web_services/foxx/brightid/initdb.js:216]: It's done.

File: updater/sponsorships.py (hunk 2 above)
Comment (siftal) [reply updater/sponsorships.py]: It's done

File: updater/sponsorships.py (hunk 3 above)
Comment (siftal) [reply updater/sponsorships.py:76]: It's done

**Review by adamstallard — APPROVED** [review]

**Pull request merged**